Update tutorial-virtual-network.md

boykoc opened the pull request:
Add some clarifying text introducing the list of default rules that are applied to a Network Security Group (NSG) when it is created.

When reading this, I was confused at first about what information this list was showing.

azurecla commented:
Hi __@SSalahAhmed__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

boykoc committed:
Update tutorial-virtual-network.md

Add some clarifying text introducing the list of default rules that are applied to a Network Security Group (NSG) when it is created.

When reading this, I was confused at first about what information this list was showing.
